<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpServer</class>
 <widget class="QWidget" name="TcpServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>121</width>
     <height>321</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>170</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="QLabel" name="labListenIP">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>监听地址</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cboxListenIP">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labListenPort">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>监听端口</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtListenPort">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnListen">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>监听</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnSave">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnClear">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="txtMain">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>331</width>
     <height>321</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layTcpServer">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QComboBox" name="cboxData">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnSend">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
